src/blocks/store/s3.clj

```diff
@@ -303,16 +313,25 @@
       (dissoc opts :credentials)
       {:client (get-client opts)
        :bucket (str/trim bucket)
-       :prefix (some-> (trim-slashes (:prefix opts)) (str "/"))})))
+       :prefix (some-> (trim-slashes (:prefix opts)) (str "/"))
+       :set-sse-flag? (boolean (:set-sse-flag? opts))})))
 
 
+(defn- parse-boolean-query-flag
```

Comment:
Per the above suggestion about making this a keyword, I don't think you need this function.